report/assets/styles.css

```diff
@@ -1066,6 +1065,10 @@
   margin-top: var(--audit-group-padding-vertical);
 }
 
+.lh-audit-group--metrics > .lh-audit-group__header {
+  visibility: hidden; /* Superfluous, because obviously perf metrics */
```

Comment:
I don't understand this... if this is not needed, why have?

Reply:
The title of "Metrics" is removed in the mock. It's one of the items in the FR Design Notes doc list.

This DOM element is added when when perf renderer calls the generic `renderAuditGroup`. so.. trying to omit the element would be a bit more awkward.

also its vis hidden (and not display:none) because spacing-wise it's still needed thanks to the PILL. stupid pill.